network-api/networkapi/mozfest/templates/mozfest/404.html

```diff
@@ -21,6 +21,7 @@
                             <div class="row">
                                 <div class="col">
                                     <div class="nav-links pt-3">
+                                        <!-- FIXME:TODO: what to do with these en links? -->
                                         <div><a class="active" href="https://www.mozillafestival.org/en/">Home</a></div>
```

Comment:
we can remove the `/en` for both of these, as mozillafestival.org is served by the same wagtail instance, just over a different domain binding.